TwilioChatClient/0.17.2/TwilioChatClient.podspec

```diff
@@ -0,0 +1,17 @@
+Pod::Spec.new do |s|
+  s.name         = "TwilioChatClient"
+  s.version      = "0.17.2"
+  s.summary      = "Twilio Programmable Chat Client"
+  s.description  = "Twilio Programmable Chat Client"
+  s.homepage     = "http://www.twilio.com/client"
```

Comment:
this should be around for a while, but wouldn't it be nice to have `https://www.twilio.com/chat`? :p